src/reporter/delete.py

```diff
@@ -34,15 +34,15 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table and fiware_sp() is None:
+        if drop_table and fiware_sp() == '/':
             trans.drop_table(etype=entity_type,
                              fiware_service=fiware_s())
             logging.getLogger(__name__).info(
                 "dropped entity_type {}".format(entity_type))
             return 'entity table dropped', 204
         elif drop_table:
             e = fiware_sp().split(",")
-            if e is not None and len(e) > 0:
+            if e != '/' and len(e) > 0:
```

Comment:
I think there's a little problem with this line. `e` is a list not a string?

```python
>>> e = "/".split(",")
>>> e
['/']
>>> e != '/'
True
```

It looks like the logic should rather be (pseudo code)

    if len(e) > 1
       error 'too many svc paths'
    elif len == 0 or e[1] != '/'
       error 'can only drop if svc path = /'
    else
       drop table

@chicco785 is that right?
@pooja1pathak if @chicco785 confirms, then can we add some tests to check the branching logic?
thanks guys!!

Reply:
yep, not sure either why we had those extra checks to start with. to me too, it'd be much better to just do what you're suggesting

```
if drop and fiware_sp() == '/':
   drop table
else:
   error ....
```

@pooja1pathak thoughts?

Reply:
if drop_table and fiware_sp() == '/':
   drop!
elif drop_table:
    e = fiware_sp().split(",")
    if  e.count(e[0]) == len(e) and e[0] == '/':
       drop!
    else:
        return "dropTable requires the root fiware-servicepath: '/'", 422

Reply:
@c0c0n3 I agree with @chicco785 's suggestion. I will update the code as per above logic.

Reply:
cool, bananas! Just one thing though. Isn't the above equivalent to this? that is @chicco785's initial suggestion:

```
if drop_table and fiware_sp() == '/':
   drop_logic
elif drop_table:
        return "dropTable requires the root fiware-servicepath: '/'", 422
```

I mean if you get into the second branch of the if statement, then you know `dropTable == True`, so it must be the case that `fiware_sp() != '/'` otherwise the first if statement would've been true? Am I missing something?

Reply:
@chicco785 @pooja1pathak sorry ignore my comment above. I wrote that too quickly. I think @chicco785 wants to drop the table also in the case there's multiple service paths but the first one is `/`. E.g. if we get

```
/v2/...?dropTable=true
fiware-servicepath: /,/other
```

then we should still drop the table. @chicco785 is that right? or have I misunderstood the intention?

Reply:
may idea was if we get `/, /other` we SHOULD NOT drop the table, if we get `/, /, /` we SHOULD drop the table

Reply:
it got inspired for a quick solution to check that all elements in the split array are equal from this discussion:
https://stackoverflow.com/questions/3844801/check-if-all-elements-in-a-list-are-identical

But it may be wrong :)

Reply:
I have updated the PR as suggested